powerline_git.lua

```diff
@@ -131,6 +182,9 @@ local function addAddonSegment()
     if segment.isNeeded then 
         addSegment(segment.text, segment.textColor, segment.fillColor)
     end 
+    if statusSegment.isNeeded then 
+        addSegment(statusSegment.text, statusSegment.textColor, statusSegment.fillColor)
+    end 
```

Comment:
> leaving the git directory does not remove the segment. not sure how to fix this myself but if i figure it out i'll submit a pull request

if this change is made, segment is removed when changing the directory

```suggestion
   if segment.isNeeded then 
        addSegment(segment.text, segment.textColor, segment.fillColor)
        if statusSegment.isNeeded then 
          addSegment(statusSegment.text, statusSegment.textColor, statusSegment.fillColor)
        end
    end 
```